frontend/src/containers/SurveyPage/SurveyPage.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect, useRef } from "react";
+import { useNavigate } from "react-router-dom";
+import { Button, Container, Typography, TextField } from "@mui/material";
+import axios from "axios";
+
+interface SurveyQuestion {
+  id: number;
+  question_text: string;
+}
+
+interface SurveyPageProps {
+  user_type: string;
+}
+
+const SurveyPage: React.FC<SurveyPageProps> = ({ user_type }) => {
+  const [surveyQuestions, setSurveyQuestions] = useState<SurveyQuestion[] | null>(null);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get(`http://localhost:5001/api/v1/survey_questions?user_type=${user_type}`);
```

Comment:
Let's use axios instance so we don't hard code the uri here. 

```
import axiosInstance from '../../axiosConfig'

const  response = await axiosInstance.get(`api/v1/survey_questions?user_type=${user_type}`)
```

Reply:
For local dev, create `.env` and add `REACT_APP_API_BASE_URL=http://localhost:5001`